app/install.php

```diff
@@ -396,6 +396,7 @@ function printStep1() {
 	<noscript><p class="alert alert-warn"><span class="alert-head"><?= _t('gen.short.attention') ?></span> <?= _t('install.javascript_is_better') ?></p></noscript>
 
 	<?php
+	<?php $version = curl_version(); ?>
```

Comment:
```suggestion
	$version = curl_version();
```

Reply:
Arf, I was to eager to go back on the theme-as-extension feature!
Fixed.